doc/guides/env_vars.rst

```diff
@@ -0,0 +1,157 @@
+-- otherwise, you have to run RapidEE as administrator. Also make sure to save
+your changes by clicking the Save button at top left before exiting.Settings
+you make in RapidEE will be available whenever you open a new terminal window.
+
+.. _env_vars_zephyrrc:
+
+Option 3: Using ``zephyrrc`` files
+----------------------------------
+
+Choose this option if you don't want to make the variable's setting available
+to all of your terminals, but still want to save the value for loading into
+your environment when you are using Zephyr.
+
+**macOS and Linux**:
+
+Create a file named :file:`~/.zephyrrc` if it doesn't exist, then add this line
+to it:
+
+.. code-block:: console
+
+   export MY_VARIABLE=foo
+
+To get this value back into your current terminal environment, **you must run**
+``source zephyr-env.sh`` from the main ``zephyr`` repository. Among other
+things, this script sources :file:`~/.zephyrrc`.
+
+The value will be lost if you close the window, etc.; run ``source
+zephyr-env.sh`` again to get it back.
+
+**Windows**:
+
+Add the line ``set MY_VARIABLE=foo`` to the file
+:file:`%userprofile%\\zephyrrc.cmd` using a text editor such as Notepad to save
+the value.
+
+To get this value back into your current terminal environment, **you must run**
+``zephyr-env.cmd`` in a ``cmd.exe`` window after changing directory to the main
+``zephyr`` repository.  Among other things, this script runs
+:file:`%userprofile%\\zephyrrc.cmd`.
+
+The value will be lost if you close the window, etc.; run ``zephyr-env.cmd``
+again to get it back.
+
+.. _zephyr-env:
+
+Zephyr Environment Scripts
+**************************
+
+You can use the zephyr repository scripts ``zephyr-env.sh`` (for macOS and
+Linux) and ``zephyr-env.cmd`` (for Windows) to load Zephyr-specific settings
+into your current terminal's environment. To do so, run this command from the
+zephyr repository::
+
+  # macOS and Linux
+  source zephyr-env.sh
+
+  # Windows
+  zephyr-env.cmd
+
+These scripts:
+
+- set :envvar:`ZEPHYR_BASE` (see below) to the location of the zephyr
+  repository
+- adds some Zephyr-specific locations (such as zephyr's :file:`scripts`
+  directory) to your :envvar:`PATH` environment variable
+- loads any settings from the ``zephyrrc`` files described above in
+  :ref:`env_vars_zephyrrc`.
+
+You can thus use them any time you need any of these settings.
+
+.. _env_vars_important:
+
+Important Environment Variables
+*******************************
+
+Here are some important environment variables and what they contain. This is
+not a comprehensive index to the environment variables which affect Zephyr's
+behavior.
+
+- :envvar:`BOARD`: allows set the board when building an application; see
+  :ref:`important-build-vars`.
+- :envvar:`CONF_FILE`: allows adding Kconfig fragments to an application build;
+  see :ref:`important-build-vars`.
+- :envvar:`DTC_OVERLAY_FILE`: allows adding devicetree overlays to an
+  application build; see :ref:`important-build-vars`.
```

Comment:
just to make this even worse, then `DTC_OVERLAY_FILE` is not being picked up from environment.
It can only be set as a CMake variable (cache / local).

Reply:
>  then `DTC_OVERLAY_FILE` is not being picked up from environment.

Sorry, what do you mean by "then"? What is the "if"?

Reply:
a stray `n`, should have been:
> the DTC_OVERLAY_FILE is not being picked up from environment.